index.html

```diff
@@ -233,11 +233,11 @@ <h2>Scope</h2>
 
             <div class="note">
                 <p class="draft-status">
-                    See also some publications for <a href="https://json-ld.github.io/rdf-dataset-canonicalization/spec/index.html">RDF Dataset Canonicalization</a> that will be used as a mathematical underpinning of the work:
+                    See also some publications for <a href="https://json-ld.github.io/rdf-dataset-canonicalization/spec/index.html">RDF Dataset Canonicalization</a> that will serve as inputs for the specification work:
```

Comment:
I would remove the link to the json-ld.github.io document here. It is not relevant to the text and it gives the (false) impression that the json-ld document is somehow the official document.